docs/prms/responsenet.rst

```diff
@@ -0,0 +1,11 @@
+ResponseNet
+===========
+
+ResponseNet is a pathway reconstruction algorithm which finds subnetworks using integer linear programming,
```

Comment:
I'm I reading the [source](https://github.com/Reed-CompBio/ResponseNet/blob/1eb9b2685af218a28af38dbcda6e5a5abca8d682/responsenet.py#L281) correctly, it uses linear programming, not integer linear programming. Is that correct? GLOP as described [here](https://developers.google.com/optimization/lp/lp_example) appears to be an LP solver.